test/integration/targets/pids/tasks/main.yml

```diff
@@ -78,5 +78,5 @@
   assert:
     that:
     - "pids.pids | length > 0"
-    - "pids.pids | join(' ')== find_process"
+    - "pids.pids | join(' ')== find_process.stdout"
```

Comment:
There's an extra space at the end of `find_process.stdout` for some of the tests. You need to trim that off, for example using jinja2's [trim filter](http://jinja.pocoo.org/docs/2.10/templates/#trim): `"pids.pids | join(' ')== find_process.stdout | trim"`